lib/path/index.js

```diff
@@ -0,0 +1,7 @@
+import _curry2 from '../_internal/_curry2'
+
+export default _curry2(function path(arr, obj) {
+  return arr.length > 1 && obj[arr[0]] !== undefined
+    ? path(arr.slice(1), obj[arr[0]])
```

Comment:
it's ineffective I suppose, because every iteration has a new allocation of array using `slice`

Reply:
Hmm, what's your proposal? Can't simply come up with another variant )

Reply:
Yeah, totally forgot to advise you to use `for(;;)` with `var state = obj` which saves current pointer to the result